spec/use_cases/esa_feeder_spec.rb

```diff
@@ -4,28 +4,36 @@
 
 RSpec.describe EsaFeeder::UseCases::Feed do
   let(:esa_client) { double('esa client') }
-
-  let(:templates) { build_list(:esa_template, 2) }
+  let(:mon_templates) { build_list(:esa_template, 2) }
+  let(:wday_templates) { build_list(:esa_template, 2, tags: %w[tag feed_wday]) }
+  let(:templates) { mon_templates + wday_templates }
   let(:posts) do
-    build_list(:esa_post, 2, tags: %w[hoge feed_mon fuga slack_test])
+    build_list(:esa_post, 4, tags: %w[hoge feed_mon fuga slack_test])
   end
 
   let(:expected) do
     [
       { templates[0].number => posts[0].number },
-      { templates[1].number => posts[1].number }
+      { templates[1].number => posts[1].number },
+      { wday_templates[0].number => posts[2].number },
+      { wday_templates[1].number => posts[3].number }
     ]
   end
 
   subject do
-    described_class.new(esa_client, slack_client).call('feed_mon', 'esa_bot')
+    described_class.new(esa_client, slack_client).call(['feed_mon', 'feed_wday'], 'esa_bot')
```

Comment:
__[RuboCop]__

Style/WordArray: Use `%w` or `%W` for an array of words.


<!-- Comment by [SideCI](https://sideci.com) -->